ReactiveCocoa/UIKit/UIGestureRecognizer.swift

```diff
@@ -0,0 +1,27 @@
+import ReactiveSwift
+import UIKit
+import enum Result.NoError
+
+extension Reactive where Base: UIGestureRecognizer {
+	/// Create a signal which sends a `next` event for each gesture event
+	///
+	/// - returns:
+	///   A trigger signal.
+	public var stateChanged: Signal<Base, NoError> {
+		return Signal { observer in
+			let receiver = CocoaTarget<Base>(observer, transform: { gestureRecognizer in
```

Comment:
You can use trailing closure here. 😸 
_Thought I have commented this before, but can't find it somehow, huh._